scripts/test.ps1

```diff
@@ -193,7 +198,30 @@ if ($NoProgress) {
     $TestArguments += " -NoProgress"
 }
 
+if ($EnableAppVerifier) {
+    # Enable Application Verifier for test binaries
+    where.exe appverif.exe
+    if ($LastExitCode -eq 0) {
+        appverif.exe /verify msquiccoretest.exe
+        appverif.exe /verify msquicplatformtest.exe
+        appverif.exe /verify msquictest.exe
+    } else {
+        Write-Debug "Application Verifier not installed!"
+        $EnableAppVerifier = $false;
+    }
+}
```

Comment:
I feel like it makes more sense to put this into the run-gtest.ps1 script (and probably run-executable.ps1 script).

Reply:
That will add about ~4 seconds to each invocation however...

Reply:
So, 4 seconds for each EXE, not per test case. The total time should be unchanged right? We're just moving setting all three in batch here, to doing it individually in run-gtest.ps1.